src/ott/geometry/regularizers.py

```diff
@@ -363,31 +379,31 @@ class STVS(ProximalOperator):
   The operator is defined as:
 
   .. math::
-    \lambda^2 \mathbf{1}_d^T \left(\sigma(x) -
+    \gamma^2 \mathbf{1}_d^T \left(\sigma(x) -
     \frac{1}{2} \exp\left(-2\sigma(x)\right) + \frac{1}{2}\right)
 
-  where :math:`\sigma(x) := \text{asinh}\left(\frac{x}{2\lambda}\right)`.
+  where :math:`\sigma(x) := \text{asinh}\left(\frac{x}{2\gamma}\right)`.
 
   Args:
-    lam: Strength of the regularization.
+    gamma: Strength of the regularization.
   """  # noqa: E501
 
-  def __init__(self, lam: float = 1.0):
+  def __init__(self, gamma: float = 1.0):
     super().__init__()
-    self.lam = lam
+    self.gamma = gamma
 
   def __call__(self, x: jnp.ndarray) -> float:  # noqa: D102
-    u = jnp.arcsinh(jnp.abs(x) / (2 * self.lam))
-    y = u - 0.5 * jnp.exp(-2.0 * u)
     # Lemma 2.1 of `schreck:15`
-    return self.lam ** 2 * jnp.sum(y + 0.5)  # make positive
+    u = jnp.arcsinh(jnp.abs(x) / (2.0 * self.gamma))
+    y = u - 0.5 * jnp.exp(-2.0 * u)
+    return self.gamma ** 2 * jnp.sum(y + 0.5)  # make positive
 
   def prox(self, v: jnp.ndarray, tau: float = 1.0) -> jnp.ndarray:  # noqa: D102
-    tmp = 1.0 - (self.lam * tau / (jnp.abs(v) + 1e-12)) ** 2
-    return jax.nn.relu(tmp) * v
+    tmp = 1.0 - ((tau * self.gamma) / (v + 1e-12)) ** 2
+    return v * jax.nn.relu(tmp)
```

Comment:
not coming from your PR, but I find this `1e-12` a bit annoying now :) 

essentially, writing `s = (tau * self.gamma)`

we have that 

```
v * relu(1 - s/ v**2) = sign(v) * |v| * relu( 1- s/v**2) = sign(v) * relu( |v| - s|v|/v**2)
= sign(v) * relu( |v| - s/|v|) 
```
the Relu is active when

```
|v|-s/|v| >0  <=> v**2 > s
```
so maybe what we could have is

```
s = (tau * self.gamma)
return jnp.where(v**2 <= s, 0, v- s/v)
```

which is kind of agreeing quite nicely with the curve.

Reply:
Unfortunately, both produce NaNs when `v==0`, tested using
```python
jax.grad(lambda x: regularizers.STVS(1).prox(x).sum())(jnp.zeros(10))
```

Reply:
Done!